care/facility/api/viewsets/patient.py

```diff
@@ -64,7 +64,7 @@ def get_queryset(self):
         filter_query = self.request.query_params.get("disease_status")
         queryset = super().get_queryset()
         if filter_query:
-            disease_status = filter_query if filter_query.isdigit() else DiseaseStatusEnum[filter_query].value
+            disease_status = filter_query if filter_query.isdigit() else DISEASE_CHOICES_VALUES[filter_query].value
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Value 'DISEASE_CHOICES_VALUES' is unsubscriptable](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5316739)